:bookmark: Release 2.8.0 (#409)
maykinmedia/open-klant: CHANGELOG.rst

Change history
==============

2.8.0
=====

> *May, 1, 2025*

Reviewer: Reminder to change this to when we actually do the release
```suggestion
*May 6, 2025*
```

**New features**

> * [:open-klant:`320`] Add ``DigitaalAdres.referentie``

Reviewer: ```suggestion
* [:open-klant:`320`] Add ``DigitaalAdres.referentie`` and allow filtering on this attribute with the `referentie` query parameter
```

* [:open-klant:`368`] Fix validation for phone numbers

> * [:open-klant:`240`] Fix optional nullable fields in creation endpoints

Reviewer: ```suggestion
* [:open-klant:`240`] Make nullable fields optional for all endpoints
```

> * [:open-klant:`342`] Add ``klantcontact``, ``betrokkene`` and ``digitaleAdressen`` partij identificator filters

Reviewer: ```suggestion
* [:open-klant:`342`] Add PartijIdentificator filters to ``/klantcontacten``, ``/betrokkenen`` and ``/digitaleadressen``
```

> * [:open-klant:`391`] Fix ``huisnummer`` nullable

Reviewer: ```suggestion
* [:open-klant:`391`] Make ``huisnummer`` nullable via the API
```

**Bugfixes**

> * [:open-klant:`378`] Fix bug in admin ``Partij`` and improve admin performance

Reviewer: ```suggestion
* [:open-klant:`378`] Fix bug that occurred when trying to create a ``Partij`` via the admin interface and improve admin performance
```

> * [:open-klant:`341`] Fix ``help_text`` in FilterSet

Reviewer: ```suggestion
* [:open-klant:`341`] Add missing help texts for several query parameters in the API schema
```

> * [:open-klant:`401`] Fix PATCH requests for ``digitaleAdressen`` and ``rekeningnummers``

Reviewer: ```suggestion
* [:open-klant:`401`] Fix PATCH requests on `/partijen/{uuid}` if ``digitaleAdressen`` and/or ``rekeningnummers`` are set to ``null``
```

> * [:open-klant:`345`] Fix PATCH requests on ``Partij`` require ``soortPartij``

Reviewer: ```suggestion
* [:open-klant:`345`] No longer make ``Partij.soortPartij`` required for PATCH requests

**Project maintenance**

> * [:open-klant:`385`] Improve performance of several endpoints

Reviewer: Might be good to list which endpoints had performance fixes

> * Upgrade commonground-api-common to 2.6.2

Reviewer: Could you also mention the NPM dependency here?
